layout.js

```diff
@@ -0,0 +1,792 @@
+        ltSpecial: '>',
+      },
+    },
+    {
+      key: 'Slash',
+      type: 'symbol',
+      text: {
+        en: '/',
+        enCapital: '/',
+        lt: '/',
+        ltCapital: '/',
+        enSpecial: '?',
+        ltSpecial: '?',
+      },
+    },
+    {
+      key: 'ArrowUp',
+      type: 'function',
+      text: {
+        en: '↑',
+        enCapital: '↑',
+        lt: '↑',
+        ltCapital: '↑',
+        enSpecial: '↑',
+        ltSpecial: '↑',
+      },
+    },
+    {
+      key: 'ShiftRight',
+      type: 'function',
+      text: {
+        en: 'Shift',
+        enCapital: 'Shift',
+        lt: 'Shift',
+        ltCapital: 'Shift',
+        enSpecial: 'Shift',
+        ltSpecial: 'Shift',
+      },
+      width: 'width-double',
+    },
+  ],
+  [
+    {
+      key: 'ControlLeft',
+      type: 'function',
+      text: {
+        en: 'Ctrl',
+        enCapital: 'Ctrl',
+        lt: 'Ctrl',
+        ltCapital: 'Ctrl',
+        enSpecial: 'Ctrl',
+        ltSpecial: 'Ctrl',
+      },
+      width: 'width-2',
+    },
+    {
+      key: 'MetaLeft',
+      type: 'function',
+      text: {
+        en: '⊞',
+        enCapital: '⊞',
+        lt: '⊞',
+        ltCapital: '⊞',
+        enSpecial: '⊞',
+        ltSpecial: '⊞',
+      },
+    },
+    {
+      key: 'AltLeft',
+      type: 'function',
+      text: {
+        en: 'Alt',
+        enCapital: 'Alt',
+        lt: 'Alt',
+        ltCapital: 'Alt',
+        enSpecial: 'Alt',
+        ltSpecial: 'Alt',
+      },
+    },
+    {
+      key: 'Space',
+      type: 'symbol',
+      text: {
+        en: ' ',
+        enCapital: ' ',
+        lt: ' ',
+        ltCapital: ' ',
+        enSpecial: ' ',
+        ltSpecial: ' ',
+      },
+      width: 'extra-wide',
+    },
+    {
+      key: 'AltRight',
+      type: 'function',
+      text: {
+        en: 'Alt',
+        enCapital: 'Alt',
+        lt: 'Alt',
+        ltCapital: 'Alt',
+        enSpecial: 'Alt',
+        ltSpecial: 'Alt',
+      },
+    },
+    {
+      key: 'ArrowLeft',
+      type: 'function',
+      text: {
+        en: '←',
+        enCapital: '←',
+        lt: '←',
+        ltCapital: '←',
+        enSpecial: '←',
+        ltSpecial: '←',
+      },
+    },
+    {
+      key: 'ArrowDown',
+      type: 'function',
+      text: {
+        en: '↓',
+        enCapital: '↓',
+        lt: '↓',
+        ltCapital: '↓',
+        enSpecial: '↓',
+        ltSpecial: '↓',
+      },
+    },
+    {
+      key: 'ArrowRight',
+      type: 'function',
+      text: {
+        en: '→',
+        enCapital: '→',
+        lt: '→',
+        ltCapital: '→',
+        enSpecial: '→',
+        ltSpecial: '→',
+      },
+    },
+    {
+      key: 'ControlRight',
+      type: 'function',
+      text: {
+        en: 'Ctrl',
+        enCapital: 'Ctrl',
+        lt: 'Ctrl',
+        ltCapital: 'Ctrl',
+        enSpecial: 'Ctrl',
+        ltSpecial: 'Ctrl',
+      },
+      width: 'width-2',
+    },
+  ],
+];
+
+export default layout;
```

Comment:
I love your approach of moving this to a separate file. Btw, I see that you are duplicating a lot of keys, a suggestion would be to add optional fields for fields that are duplicated. For example, when you need `enSpecial`, and it is null (when it's duplicated considering the `en` version), you just take the `en` version instead